src/drive/web/modules/views/OnlyOffice/Title.jsx

```diff
@@ -21,6 +21,8 @@ const Title = () => {
   const sharingInfos = useSharingInfos()
   const styles = useStyles()
 
+  const showSharingBanner = isPublic && window.history.length <= 1
```

Comment:
can you explain why this history.length - 1? Is it when we arrive on an OO document from a sharing link, but the sharing link is only for this document? 

I don't understand why we don't display the "SharingBanner" if no history, sorry :/

Reply:
The sharing banner need to be shown only on the first arrival not after browsing somewhere, inside a folder for example